Uplift/Supporting/Constants.swift

```diff
@@ -35,6 +35,14 @@ struct GymIds {
     static let teagleUp = "939c7a2c16d2299cc8558475a8007defc414069c"
 }
 
+// MARK: - CLASS IDS
+struct ClassIds {
+    static let yogaVinyasa = "057794a5c10fb355cc5c31a0989781b3271f03f7"
+    static let cuRowShockwave = "13dacafc5638331cc9c4477b918c914db3c1310b"
+    static let zumba = "eb318ac64f7eab2046aa38ad52ca487b11c006a8"
+    static let musclePump = "73e8b1961738da4ba44efd45a925b61bf59be760"
+}
```

Comment:
why are we hard coding these? i feel like this should be something we fetch from backend

Reply:
I included those for when the Onboarding can't connect to internet, and creates 4 default classes with a name and id to set as a favorite